tests/image_test.py

```diff
@@ -1224,6 +1225,23 @@ def test_load_img(self, tmpdir):
             loaded_im = image.load_img(filename_rgb, target_size=(25, 25),
                                        interpolation="unsupported")
 
+    def test_list_pictures(self, tmpdir):
+        filenames = ['test.png', 'test0.jpg', 'test-1.jpeg', '2test.bmp',
+                     '2-test.ppm', '3.png', '1.jpeg', 'test.bmp', 'test0.ppm']
```

Comment:
add `.tif` and `.tiff` filenames, and a couple of filenames that should not be retrieved by the function, than change the assertions accordingly.